### ISSUE TITLE
Deployandtest fails tests for "metal" deploy

### ISSUE THREAD

**Issue by mandoonandy** [shakenfist/deploy#15]
```
TASK [Extract instance IP] **************************************************************************************************************************************************************
ok: [sf-1]
ok: [sf-2]
ok: [sf-3]

TASK [Log instance details] *************************************************************************************************************************************************************
ok: [sf-1] => {
    "msg": "Instance cirros has uuid 57475d3e-ff7f-4c02-a356-1c7e5cacb0fe, console port 37785, and IPv4 192.168.242.202."
}
ok: [sf-2] => {
    "msg": "Instance cirros has uuid 504b3b4c-f568-4ea4-b2b6-1559c382e3c2, console port 39267, and IPv4 192.168.242.70."
}
ok: [sf-3] => {
    "msg": "Instance cirros has uuid 40e61048-90a3-452f-99b5-479b339c7765, console port 48817, and IPv4 192.168.242.225."
}

TASK [Wait for instance to boot] ********************************************************************************************************************************************************
Pausing for 120 seconds
(ctrl+C then 'C' = continue early, ctrl+C then 'A' = abort)
ok: [sf-1]

TASK [Validate] *************************************************************************************************************************************************************************
changed: [sf-2]
changed: [sf-3]
changed: [sf-1]

TASK [Execute] **************************************************************************************************************************************************************************
fatal: [sf-2]: FAILED! => {"changed": true, "cmd": "/tmp/shell_script", "delta": "0:00:01.329107", "end": "2020-07-09 01:23:25.975007", "msg": "non-zero return code", "rc": 1, "start": "2020-07-09 01:23:24.645900", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
fatal: [sf-3]: FAILED! => {"changed": true, "cmd": "/tmp/shell_script", "delta": "0:00:01.291651", "end": "2020-07-09 01:23:26.120864", "msg": "non-zero return code", "rc": 1, "start": "2020-07-09 01:23:24.829213", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
fatal: [sf-1]: FAILED! => {"changed": true, "cmd": "/tmp/shell_script", "delta": "0:00:01.341354", "end": "2020-07-09 01:23:26.570787", "msg": "non-zero return code", "rc": 1, "start": "2020-07-09 01:23:25.229433", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
```

**Comment by mikalstill** [comment]
I will run a test on this before release.

**Comment by mikalstill** [comment]
So this is working for me, but it might be a flakey test. These ansible based CI tests are meant to be going away, but it isn't quite done yet...

**Comment by mandoonandy** [comment]
Works for me now too.
ESXi needs virtualised CPU performance counters enabled. This due to a bug in KVM doing an incomplete check. This fix was committed 2020-02-11. Not in latest release.
https://communities.vmware.com/thread/592140
https://bugs.launchpad.net/qemu/+bug/1661386

TLDR; ESXi needs two settings enabled in the VM --> I'll add that to deploy README

KUDOS: The deploy tests detected the problem
